misc/install-deps.sh

```diff
@@ -14,20 +14,25 @@ distro=$(grep "^ID=" /etc/os-release | cut -d\= -f2 | sed -e 's/"//g')
 case $distro in
     "ubuntu" | "debian")
         apt-get $OPT install pandoc libdw-dev libpython2.7-dev libncursesw5-dev pkg-config
-        apt-get $OPT install libcapstone-dev libluajit-5.1-dev ;;
+        apt-get $OPT install libluajit-5.1-dev || true
+        apt-get $OPT install libcapstone-dev || true ;;
     "fedora")
         dnf install $OPT pandoc elfutils-devel python2-devel ncurses-devel pkgconf-pkg-config
-        dnf install $OPT capstone-devel luajit-devel ;;
+        dnf install $OPT luajit-devel || true
+        dnf install $OPT capstone-devel || true ;;
     "rhel" | "centos")
         rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
         yum install $OPT pandoc elfutils-devel python2-devel ncurses-devel pkgconfig
-        yum install $OPT capstone-devel luajit-devel ;;
+        yum install $OPT luajit-devel || true
+        yum install $OPT capstone-devel || true ;;
     "arch" | "manjaro")
-        pacman $OPT -S pandoc libelf python2 ncurses pkgconf
-        pacman $OPT -S capstone luajit ;;
+	pacman $OPT -S pandoc libelf python2 ncurses pkgconf
+	pacman $OPT -S luajit || true
+	pacman $OPT -S capstone || true ;;
```

Comment:
Please fix the indentation, we used all spaces here.

Reply:
Thanks. I fixed.